### ISSUE TITLE
After resizing one moveable component, other components cannot be draggable

### ISSUE THREAD

**Issue by Kun8018** [daybrush/moveable#769]
## Environments
* Framework name: React
* Framework version: 16
* Moveable Component version: 0.39
* Testable Address(optional):

## Description
After resizing one moveable component, other components cannot be draggable

Code like 
`  const useCustomRef = [
    React.useRef(null),
    React.useRef(null),
  ];

  const target = [1, 2];

  return (
    <>
      {target.map((_, index) => {
        return (
          <>
            <div
              className="target"
              ref={useLottieRef[index]}
              style={{
                transform: 'translate(0px, 0px) rotate(0deg) scale(1, 1)',
                display: 'inline-block',
              }}
            >
              Target{index}
            </div>
            <Moveable  ..../>
      )}}
     </>
  )
`

**Comment by daybrush** [comment]
@Kun8018 

Can you tell me how to use your moveable?

**Comment by daybrush** [comment]
@Kun8018 

sorry. Moveable is optimized for absolute.

In the case of inline-block and position: relative, all other moveables are affected.

Unfortunately, there is no Browser API that can automate this part.

You have to manually call moveable.updateRect().

**Comment by Kun8018** [comment]
> @Kun8018
> 
> sorry. Moveable is optimized for absolute.
> 
> In the case of inline-block and position: relative, all other moveables are affected.
> 
> Unfortunately, there is no Browser API that can automate this part.
> 
> You have to manually call moveable.updateRect().

Yes, i think you have known what i mean. There are several moveable components in dom. After i resize the first component, the second one and the third movable component will be influenced to be relocated and can't be draggable unless i click the dom inside the moveable component. 
Now i use only one moveable component and update the target thus will not reproduce the bug. I thinks It's better to use absolute position for every moveable component

**Comment by Kun8018** [comment]
error code like:
`import { moveable } from react-moveable;

const App = () => {
   return <>
     <div className="target1">target1</div>
     <div className="target2">target2</div>
      <moveable target={target1}.../> 
      <moveable target={target2} ..../> 
   <>
}
 `
Now my code like 
`import { moveable } from react-moveable;
import { useState } from react;

const App = () => {
   const [targetState, setTargetState] = useState(target1);
   return <>
     <div className="target1">target1</div>
     <div className="target2">target2</div>
      <moveable target={targetState}.../> 
   <>
}
 `

**Comment by Kun8018** [comment]
I delete the display: inline-block css property and add positon: absolute, it works to me.

By the way, I wanna ask if i can use the native h5 api onDragover to detect the collision between the drag component? Or how can i realize the similar function? thanks.

**Comment by daybrush** [comment]
@Kun8018 

sorry. DND API is not available. Since target is moved, it cannot be dropped.

However, if it is a collision check, there is a method `moveable.hitTest`.

If you do moveable.hitTest(element), you can check whether it has collided. (100% = 100)
